### ISSUE TITLE
Openshot Video Editor does not recognize Blender

### ISSUE THREAD

**Issue by AlanReviews** [OpenShot/openshot-qt#4793]
**Describe the bug:**
My version of Blender is 3.0.1. I configured the path of Blender to be blender or /usr/bin/blender. If the path of Blender is blender, Openshot Video Editor crashes. If I used /usr/bin/blender, I get an error message:
Blender, the free open source 3D content creation suite, is required for this action. (http://www.blender.org)

Please check the preferences in OpenShot and be sure the Blender executable is correct.
This setting should be the path of the 'blender' executable on your computer.
Also, please be sure that it is pointing to Blender version 2.80 or greater.

Blender Path: /usr/bin/blender

**Steps to reproduce the behavior:**
1. Go to edit then preferences
2. Change the Blender path to be blender
3. Create an animated title by going to title then animated title.
4. Openshot Video Editor crashes.
5. Open Openshot Video Editor again,
6. Change the path of Blender to /usr/bin/blender
7. Create an animated title by going to title then animated title.
8. You will receive an error message

**Expected behavior:**
With the Blender path configured, I can create an animated title.

**System Details:**
- OpenShot Version 2.6.1
- Operating System / Distro: Pop!_OS 22.04

**Log Files:**
[openshot-qt.zip](https://github.com/OpenShot/openshot-qt/files/8648319/openshot-qt.zip)

**Screenshots:** (Optional)

![error message](https://user-images.githubusercontent.com/54216387/167327778-8ee744e9-686a-4553-a41b-a96e4e067e79.png)

**Comment by JacksonRG** [comment]
We recently added a fix for blender for recent Linux Distros (Ubuntu 22.04 And it's derivatives included).

I'd love to hear whether this issue still exists on the latest [daily build](https://www.openshot.org/download/) of openshot.

**Comment by slyholborn** [comment]
Hi! I've got the same issue on fedora 36 with the last daily build... I tried to change the path of Blender to /usr/bin/blender but i still have this on my terminal : 
`INFO main_window: Loaded project /home/sylvain/sejour ados/broceliande.osp
INFO ui_util: Initializing UI for Dialog
INFO blender_model: updating effects model.
INFO blender_listview: Animation param fontname set to Bfont
INFO blender_listview: Animation param spacemode set to CENTER
INFO blender_listview: Animation param length_multiplier set to 2
INFO blender_listview: Animation param length_multiplier set to 1
INFO blender_listview: Previewing frame 125
Traceback (most recent call last):
  File "/tmp/.mount_ZPocVy/usr/bin/windows/views/blender_listview.py", line 864, in Render
    if not self.version and not self.blender_version_check():
  File "/tmp/.mount_ZPocVy/usr/bin/windows/views/blender_listview.py", line 810, in blender_version_check
    raise Exception("No Blender version detected in output")
Exception: No Blender version detected in output
`

What shall i do to have animated tittles available? Thanks!

**Comment by JacksonRG** [comment]
@slyholborn Could you turn on debug mode and try that again? Then could you drag your log files into a comment here?
[Here](https://github.com/OpenShot/openshot-qt/wiki/Issue-Logging) are instructions to do that.

That will give me more info on where this issue is occurring.

**Comment by JacksonRG** [comment]
@AlanReviews is there any chance you have blender installed as a flatpak or snap? Also have you tested a daily build?

In the mean time I'll try installing the PPA version, and see if my setup (very similar to yours. Pop 22.04, blender 3.1) works.

**Comment by slyholborn** [comment]
[libopenshot.log](https://github.com/OpenShot/openshot-qt/files/9070952/libopenshot.log)
[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/9070953/openshot-qt.log)

 I don't have pop-up with the daily build. It just turns in vain... My Blender version is the fedora 36 repo version.

**Comment by slyholborn** [comment]
I don't have pop-up with the daily build. It just turns in vain...

**Comment by slyholborn** [comment]
Still hapenning in last daily build...

**Comment by Bernx01** [comment]
... same problem on my Ubuntu 22.04

**Comment by slyholborn** [comment]
Hi! Still got that issue with a brand new fedora 37, Blender from fedora reps (no flatpak nor snap) and daily build (27/11)...

**Comment by Colorjet3** [comment]
Please tell us which version of OpenShot you are running (please include the build #).  Also, what is the blender path set to in Edit | Preference | General tab?

**Comment by AlanReviews** [comment]
The version I am running is 2.6.1.
My blender path is set to /usr/bin/blender

I downloaded both Blender and Openshot from the Pop shop.

**Comment by AlanReviews** [comment]
Yes, I installed both Blender and Openshot as flatpaks.

**Comment by Colorjet3** [comment]
Hello @AlanReviews . I am not an expert in Linux and not very familiar with flatpak vs. PPA vs. AppImage.  With that said, I do have OpenShot working with Blender in Ubuntu 22.04.1 and Blender version 3.3.1.  

My configuration looks like this:
1.  Running OpenShot Daily Build #10243.  Version 2.6.1-Release-Candidat is very buggy.
2.  I installed Blender via the Ubuntu's Software Updater.  Currently running version 3.3.1.
3.  In Openshot, Edit | Preferences | General tab, I have "Blender" for the "Blender Command (path)" attribute, excluding the quotes.
4.  This seems to work for me.

If you want to give the AppImage a try, go to openshot.org/download/#daily.  Download build #10243.  There are newer releases of daily builds but DO NOT download those at this time.  There are Export issues currently that is being worked on.

Run the OpenShot AppImage and see if Blender will work for you.  This should work for you.  If you are still having getting the message regarding Blender is required, then change your path to Blender to /usr/bin/blender as you reported above.  I am assuming that you were able to browse to this location and select it.

Let us know if the AppImage route worked.

**Comment by AlanReviews** [comment]
I downloaded Blender with apt. I downloaded the latest version of the Openshot video editor. I can create animated titles.

**Comment by AlanReviews** [comment]
I tried to create animated titles. I get 100% usage of my CPU. The render does finish. The UI to create the render freezes. I do not get the full animated title. I am using Blender 3.0.1 and Openshot 3.1.1 appimage. I installed Blender from snap. My operating system/distro is Pop!_OS 22.04.
![openshot video editor animated title freezing](https://github.com/OpenShot/openshot-qt/assets/54216387/c0067ead-3423-46c6-a1c7-dcdce3a177af)
